src/ICS20Transfer.sol

```diff
@@ -148,18 +145,47 @@ contract ICS20Transfer is IIBCApp, IICS20Transfer, IICS20Errors, Ownable, Reentr
     /// @inheritdoc IIBCApp
     function onTimeoutPacket(OnTimeoutPacketCallback calldata msg_) external onlyOwner nonReentrant {
         ICS20Lib.PacketDataJSON memory packetData = ICS20Lib.unmarshalJSON(msg_.packet.data);
-        (address erc20Address,) = getSendERC20AddressAndSource(msg_.packet, packetData);
-        _refundTokens(packetData, erc20Address);
+        _refundTokens(msg_.packet, packetData);
 
         emit ICS20Timeout(packetData);
     }
 
+    /// @notice Get the escrow address for a channel ID
+    /// @param channelId The channel ID
+    /// @return The address of the escrow contract
+    function _getEscrowAddress(string memory channelId) internal returns (address) {
```

Comment:
The name/natspec should explain that this function also deploys the escrow contract